_posts/2022-10-31-segmentation.md

```diff
@@ -0,0 +1,314 @@
+
+I think I have explained enough about how CellProfiler and Cellpose are both great in their own ways, but I want to finally discuss one of the biggest issues that I have with the Cellpose plugin compared to the other two methods.
+
+As I said earlier, the CellProfiler Cellpose plugin does not use any nuclei objects from a previous module to find whole cells. 
+This means there is currently no way to match any nuclei with their respective cytoplasm/whole cell.
+
+What I believe this plugin should be able to do is remove any nuclei associated with a cytoplasm that has pixels touching an edge of the image. 
+For the Cellpose method, in the projects within the lab, we created custom Python functions to make sure that any nuclei that fall in between a cytoplasm is included and any cytoplasm without nuclei is gotten rid of.
+
+With that being said, it is also good to keep in mind that the CellProfiler Cellpose plugin does not automatically connect different objects. 
+This will impact downstream processing and analysis since you won't be able to, for example, connect the cells to their nuclei without additional customization (e.g. create custom Python functions, etc.).
+
+In all, I think that the CellProfiler Cellpose plugin would work much better if it included the CellProfiler method of using segmented nuclei as the base for segmenting the whole cells or to add functionality that can connect the cytoplasm to their respective nucleus (or nuclei). 
+This will be a project that I will be pursuing in the future!
+
+## Conclusion
+
+After all of the prototyping, I have come to one conclusion: All these segmentation methods are good! 
+They are all very easy to use and seem to work well by eye. 
+It is yet to be seen if these subtle segmentation differences that I discussed will impact cell morphologies and thesubsequent biology that we discover (e.g. biomarkers for gene deficiency).
+
+I will be using all of them for various projects and not one of them is substantially better than the other. 
+They each have their pros and cons that I have talked about, where you can easily decide which one will be best for your project (Table 1). 
+I highly recommend prototyping with each of them before deciding for yourself.
+
+I hope that my testing has helped you and I will be seeing you next time when I go over feature extraction methods!
+
+{%
+  include figure.html
+  image="images/blog/segmentation/segmentation_table.png"
+  width="100%"
+%}
+
+> Table 1. Pros and cons between segmentation methods.
```

Comment:
Do I recall that you said Cellpose does in fact have a CLI?

Reply:
I think python should be row 3 (before prototyping) (super nitpicky, I know)

Reply:
@gwaybio 

For the first comment, yes it does have a CLI but I never tested it, so I put in the key for N/A that is means "not applicable, did not test". I am sure where I could put that information.

No worries on the second comment, I will edit it.